tutorials/underwater_vehicles.md

```diff
@@ -11,17 +11,12 @@ guide you through the setup of the [MBARI LRAUV](https://app.gazebosim.org/accur
 
 ## Related tutorials
 
-https://gazebosim.org/api/sim/8/create_vehicle.html
-
-https://gazebosim.org/api/sim/8/adding_visuals.html
-
-https://gazebosim.org/api/sim/8/frame_reference.html
-
-https://gazebosim.org/api/sim/8/adding_system_plugins.html
-
-https://gazebosim.org/api/sim/8/theory_buoyancy.html
-
-https://gazebosim.org/api/sim/8/theory_hydrodynamics.html
+\ref create_vehicle
+\ref adding_visuals
+\ref frame_reference
+\ref adding_system_plugins
+\ref theory_buoyancy
+\ref theory_hydrodynamics
```

Comment:
need to add new line between them otherwise these get rendered in the same line:

<img width="712" alt="Screenshot 2024-08-28 at 9 06 40 PM" src="https://github.com/user-attachments/assets/16b5da3a-cd46-4e23-b478-ef39c0914d59">